.github/workflows/publish_pypi.yml

```diff
@@ -0,0 +1,31 @@
+on: push
```

Comment:
Please change this event to "release on created mode"

Reply:
Oh, okay. I actually thought it works when a tagged commit (for release) is pushed, but I will change it to the new one now 👍